protos/bottle/fulfillment/v1/order.proto

```diff
@@ -13,6 +13,7 @@ message Order {
   bottle.account.v1.Address shipping = 3;
   bottle.account.v1.Address billing = 4;
   string shipping_method = 5;
-
+  string payment_method = 7;
```

Comment:
`stripe`, `check`, `wire_transfer`, et al.

Reply:
We might want to change this from string to [enum](https://developers.google.com/protocol-buffers/docs/proto#enum)